alloy.py

```diff
@@ -340,18 +340,20 @@ def unsupported_llvm_targets(LLVM_VERSION):
 # native - native targets run natively on current hardware.
 # sde - native target, which need to be emulated on current hardware.
 def check_targets():
+    from os.path import join
+
     result = []
     result_sde = []
-    # check what native targets do we have
-    if current_OS != "Windows":
-        if options.ispc_build_compiler == "clang":
-            cisa_compiler = "clang"
-        elif options.ispc_build_compiler == "gcc":
-            cisa_compiler = "g++"
 
-        try_do_LLVM("build check_ISA", cisa_compiler + " check_isa.cpp -o check_isa.exe", True)
-    else:
-        try_do_LLVM("build check_ISA", "cl check_isa.cpp", True)
+    cmd_conf = "cmake . -B check_isa_build -DISPC_INCLUDE_UTILS_ONLY=ON"
+    cmd_build = "cmake --build check_isa_build --target check_isa"
```

Comment:
Minor note: I think CMAKE_BUILD_TYPE is Release by default but `cmake --build` builds Debug by default. Let's specify configuration explicitly: 
```suggestion
    cmd_build = "cmake --build check_isa_build --target check_isa --config Release"
```

Reply:
> but `cmake --build` builds Debug by default

Interesting, I didn't know this. Is using `--config Release` in the build line different from specifying `-DCMAKE_BUILD_TYPE=Release` in the `cmd_conf` line?

Reply:
I think it is. Our default build type (Release) for CMake configuration is set in CMakeLists.txt: https://github.com/ispc/ispc/blob/main/CMakeLists.txt#L233 but `cmake --build` without specified configuration will do Debug build (at least on Windows).